### PR TITLE
Add some missing IEx helpers in the documentation list

### PR DESCRIPTION
**Pull request by lucazulian**
This PR adds some missing IEx helpers in the documentation.

### CODE REVIEW

**Comment by josevalim** [comment]
Thank you, however, this list is not mean to be complete. However, if you want, we can add a paragraph and a summary of shortcuts used for debugging. WDYT?

**Comment by lucazulian** [comment]
> Thank you, however, this list is not mean to be complete. However, if you want, we can add a paragraph and a summary of shortcuts used for debugging. WDYT?

I had the suspicion that the list wasn't supposed to be complete, and adding a summary of shortcuts used for debugging is a perfect idea. Is there a way I can help with it?

**Comment by josevalim** [comment]
You can change this PR by adding a new paragraph after and we take it from there. WDYT?

**Comment by josevalim** [comment]
Ping!

**Comment by lucazulian** [comment]
> Ping!

I'm so sorry. Yes, adding a new paragraph is a good idea; I will change the actual PR with it! Thanks for your patience :pray:

File: lib/iex/lib/iex/helpers.ex [hunk 1]
```diff
@@ -22,33 +22,45 @@ defmodule IEx.Helpers do
...
+
+    * `break!/2`        - sets up a breakpoint in the AST of shape `Module.function/arity` with the number of `stops`
+    * `breaks/0`        - prints all breakpoints to the terminal
+    * `c/0`             - a shortcut for `continue/0`
+    * `continue/0`      - continues execution of the current process
+    * `n/0`             - a shortcut for `next/0`
+    * `next/0`          - goes to the next line of the current breakpoint
+    * `reset_break/1`   - resets the breaks on a given breakpoint ID.
+    * `remove_breaks/0` - removes all breakpoints and instrumentation from all modules.
+    * `whereami/1`      - prints the current location and stacktrace in a pry session
```
Comment (josevalim) [inline lib/iex/lib/iex/helpers.ex:63]:
```suggestion
  There are also several helpers available when debugging, such as:

    * `break!/2`        - sets a breakpoint at `Module.function/arity`
    * `breaks/0`        - prints all breakpoints to the terminal
    * `c/0`             - a shortcut for `continue/0`
    * `continue/0`      - continues execution of the current process
    * `n/0`             - a shortcut for `next/0`
    * `next/0`          - goes to the next line of the current breakpoint
    * `remove_breaks/0` - removes all breakpoints and instrumentation from all modules
    * `whereami/1`      - prints the current location and stacktrace in a pry session
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**